semantics/language/execution/expr/function-call.k

```diff
@@ -210,6 +210,8 @@ module C-EXPR-FUNCTION-CALL
           requires fromConstantExpr(T)
      rule _:KResult := tv(_:CValue, T:Type => stripConstants(T))
           requires fromConstantExpr(T)
+     rule _:KResult := te(_, T:Type => stripConstants(T))
+          requires fromConstantExpr(T)
```

Comment:
I don't think this should be necessary -- te() => tv() in the execution semantics (see init.k:58).